code/game/objects/items/devices/instruments.dm

```diff
@@ -114,6 +114,17 @@
 	icon_state = "[initial(icon_state)]"
 	update_icon()
 
+/obj/item/instrument/piano_synth/headphones/spacepods
+	name = "nanotrasen space pods"
+	desc = "Flex your money, AND ignore what everyone else says, all at once!"
+	icon = 'icons/obj/clothing/accessories.dmi'
+	icon_state = "spacepods"
+	item_state = "spacepods"
+	slot_flags = ITEM_SLOT_EARS

```

Comment:
not needed

Reply:
probably is, since if you tried wearing it on your head it would be hidden by your headset icon

Reply:
inheritance

Reply:
i just tested it, the slot flag IS needed. otherwise the spacepods would be invisible due to being covered up by your headset, assuming you wore it on your head